packages/library/src/providers/CFDProvider/CFDProvider.tsx

```diff
@@ -39,7 +42,18 @@ export const CFDProvider = ({ children }: { children: React.ReactNode }) => {
         setCfdState(prevState => ({ ...prevState, [key]: value }));
     };
 
-    const providerValue = useMemo(() => ({ getCFDState, setCfdState: updateCFDState }), [getCFDState]);
+    const hide = useCallback(() => {
```

Comment:
What is the reason of using useCallback here ?

Reply:
I guess it isn't necessary since it doesn't depend on any prop for now, will remove it